solidity/contracts/middleware/libs/InterchainAccountMessage.sol

```diff
@@ -32,7 +32,29 @@ library InterchainAccountMessage {
     ) internal pure returns (bytes memory) {
         CallLib.Call[] memory _calls = new CallLib.Call[](1);
         _calls[0] = CallLib.build(_to, _value, _data);
-        return abi.encode(TypeCasts.addressToBytes32(_owner), _ism, _calls);
+        return
+            abi.encode(
+                TypeCasts.addressToBytes32(_owner),
+                _ism,
+                _calls,
+                bytes32(0) // Salts are expected when decoding.
```

Comment:
should we pull this out into a shared constant?

Reply:
Done